### DISCUSSION TITLE
Multiple group by fields in github project

### DISCUSSION THREAD

**Discussion by haimrait** [community/community#40951] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

It would be very beneficial to create a group by multiple fields in GitHub project.
For example, In sprint view, it would be helpful to group by Sprint field and group by the assignee.
Is it in the Roadmap?

**Comment by ossanna16** [comment]
Hi there @haimrait and welcome to our community! Thank you for the feedback!
